### PR TITLE
Javascript run nested promises parallelly

### PR DESCRIPTION
**Pull request by gomorizsolt**

### CODE REVIEW

File: .github/workflows/javascript-run-nested-promises-parallelly.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Run nested promises parallelly
```
Comment (thisismydesign) [inline .github/workflows/javascript-run-nested-promises-parallelly.yml]:
```suggestion
name: Run nested promises in parallel
```

File: javascript-run-nested-promises-parallelly/javascript-run-nested-promises-parallelly.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const randomDelay = (min, max) => Math.floor(Math.random() * (max - min) + min);
+const uniqueIdentifier = () => '_' + Math.random().toString(36).substr(2, 9);
+const delayedPromise = (delayMin, delayMax) => new Promise(resolve => setTimeout(() => resolve(uniqueIdentifier()), randomDelay(delayMin, delayMax))); 
+
+const promises = [];
+
+for (let i = 0; i < 5; i++) {
+    promises.push(delayedPromise(2000, 7000).then(outerPromiseId => {
```
Comment (thisismydesign) [inline javascript-run-nested-promises-parallelly/javascript-run-nested-promises-parallelly.js]: How about also using `new Array(3).fill()` here?

File: .github/workflows/javascript-run-nested-promises-parallelly.yml (hunk 1 above)
Comment (gomorizsolt) [reply .github/workflows/javascript-run-nested-promises-parallelly.yml]: Hmmm, thanks! Does it mean there's no word `parallelly`?

Comment (gomorizsolt) [reply .github/workflows/javascript-run-nested-promises-parallelly.yml]: 6cac08a

File: javascript-run-nested-promises-parallelly/javascript-run-nested-promises-parallelly.js (hunk 2 above)
Comment (gomorizsolt) [reply javascript-run-nested-promises-parallelly/javascript-run-nested-promises-parallelly.js]: 👍

Comment (gomorizsolt) [reply javascript-run-nested-promises-parallelly/javascript-run-nested-promises-parallelly.js]: 91847ff

File: .github/workflows/javascript-run-nested-promises-parallelly.yml (hunk 1 above)
Comment (thisismydesign) [reply .github/workflows/javascript-run-nested-promises-parallelly.yml]: I think so but not sure

File: javascript-run-nested-promises-in-parallel/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Run nested promises in parallel
+
```
Comment (thisismydesign) [inline javascript-run-nested-promises-in-parallel/README.md]: Please add a description that we want to achieve maximum parallelism and how that is visible in the logs.

Comment (gomorizsolt) [reply javascript-run-nested-promises-in-parallel/README.md]: Please take a look at the changes. That's the best wordings I could come up, proposals welcome.

**Pull request merged**